src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -7736,14 +7736,20 @@ abstract public class clx
     } // class clx
 }
 ";
-            var comp = DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(text,
-                new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractNotVirtual, Line = 5, Column = 40 },
-                new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractNotVirtual, Line = 6, Column = 41 },
-                new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractNotVirtual, Line = 7, Column = 40 },
-                new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractNotVirtual, Line = 8, Column = 53 });
 
-            var ns = comp.SourceModule.GlobalNamespace.GetMembers("NS").Single() as NamespaceSymbol;
-            // TODO...
+            var comp = CreateStandardCompilation(source).VerifyDiagnostics(
+               // (5,40): The abstract method 'clx.M1' cannot be marked virtual
```

Comment:
>// (5,40): The abstract method 'clx.M1' cannot be marked virtual [](start = 15, length = 64)

Please include comments with the line of source, see other tests that use VerifyDiagnostics helper. If you simply remove all the errors and rerun the test, the "Actual" portion of the test output will have all the right pieces.